client/src/app/apiServices.tsx

```diff
@@ -17,4 +19,17 @@ export const updateSurvey = async (id: number, updatedValue: string, value: stri
         name: 'Dogs in Scotland',
         complete: false,
         publish: false}]
+}
+
+export const sendEmail = async (userId: number, emails: string[], id: string) => {
```

Comment:
you don't have to create this method. You should use shareSurvey method that is already implemented in /services/survey.service.ts file.